pkg/plugins/resources/postgres/store_template_test.go

```diff
@@ -42,10 +42,8 @@ var _ = Describe("PostgresStore template", func() {
 		}
 	}
 
-	test_store.ExecuteStoreTests(createStore("pgx", 0), "pgx")
```

Comment:
Why this change?

Reply:
crap, was testing something and forgot to revert this, will revert - sorry!